ls1intum/Artemis
# `Development`: Use Athene Client Dependency from Maven Central

### Description
So far, we used the [Athene client](https://github.com/ls1intum/Athene/tree/master/client) as a `.jar` file stored in the `lib` directory. This is not a best practice and I eventually published the client to Maven central. This PR switches from the JAR file to a proper dependency.

### Steps for Testing
1. Create a text exercise with Automatic assessment enabled
2. Create at least 10 identical submissions with English text and multiple sentences (e.g., copy from Wikipedia)
3. Exercise Deadline
4. check cluster statistics after ~ 2min and find clusters.
4. Assess submissions
5. Find feedback suggestions starting with second assessment.
